cmd/bbolt/command_get.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	bolt "go.etcd.io/bbolt"
+)
+
+func newGetCommand() *cobra.Command {
+	var parseFormat, format string
```

Comment:
@ahrtr would this be ok, or should be in a un-exported struct ?

Reply:
I think it's better to add them into a struct, similar to https://github.com/etcd-io/bbolt/blob/6babb3faaea39276ef1d110067faef0e53b17ae9/cmd/bbolt/command_page_item.go#L16-L20